sdb/command.py

```diff
@@ -17,10 +17,38 @@
 
 import argparse
 import inspect
-from typing import Iterable, List, Optional
+from typing import Callable, Dict, Iterable, List, Optional, Type, TypeVar
 
 import drgn
-import sdb
+
+from sdb.error import CommandError, SymbolNotFoundError
+import sdb.target as target
+
+#
+# The _register_command is used by the Command class when its
+# subclasses are initialized (the classes, not the objects),
+# so we must define it here, before we import those classes below.
+#
+all_commands: Dict[str, Type["Command"]] = {}
+
+
+def register_command(name: str, class_: Type["Command"]) -> None:
+    """
+    Register the specified command name and command class, such that the
+    command will be available from the SDB REPL.
+    """
+    # pylint: disable=global-statement
+    global all_commands
+    all_commands[name] = class_
+
+
+def get_registered_commands() -> Dict[str, Type["Command"]]:
+    """
+    Return a dictionary of command names to command objects.
```

Comment:
should we be more specific and state "class objects" rather than "objects"? or perhaps just "class"? otherwise, I might interpret that to mean an instance object of the class, rather than the class itself.

Reply:
Makes sense to me